deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -123,6 +134,30 @@ spec:
             path: adminAuth.secretName
             x-descriptors:
               - 'urn:alm:descriptor:io.kubernetes:Secret'
+      - description: Batch is the Schema for the batches API
+        kind: Batch
+        name: batches.infinispan.org
+        version: v2alpha1
+        displayName: Batch
+        specDescriptors:
+          - displayName: Cluster name
+            description: Infinispan cluster name
+            path: cluster
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes:infinispan.org:v1:Infinispan'
+          - displayName: Config command
+            description: List of commands to be executed
+            path: config
+          - displayName: ConfigMap name
```

Comment:
It's possible to add alm x-descriptor for ConfigMap list to be available

Reply:
yeah done